### ISSUE TITLE
Training issue

### ISSUE THREAD

**Issue by superblaubeere27** [msiemens/HypheNN-de#2]
When training the training loss quickly drops to under `7.1927e-06` (after the first epoch). When I finished the training and predicted the output of some verbs, I noticed that the output for every character tends towards 0. The AI never decides to split at the syllable. The data is read properly. What can this be caused by?

**Comment by msiemens** [comment]
Weird. I can't reproduce this on my machine so I guess something may be broken with your Keras/Tensorflow installation… Which versions are you using? I tested using Keras 2.4.3, Tensorflow 2.3.0 and NumPy 1.18.5.

You could also try to use the word list from https://gist.github.com/msiemens/2aac63cf8d1b88c48d33c9c82f8f8e15. That's the one I used.

**Comment by superblaubeere27** [comment]
I have Keras 2.0.5, Tensorflow 1.5.0 and Numpy 1.19.1. I will try your versions. Thank you

**Comment by msiemens** [comment]
Feel free to re-open if you still have issues!

**Comment by superblaubeere27** [comment]
I can't reopen it. However, This issue consists when using Python 3.8 with TensorFlow 2.3.0, cuda 11, cuDNN 8, numpy 1.18.5 and keras 1.4.3
